taca/storage/storage.py

```diff
@@ -178,35 +179,56 @@ def cleanup_uppmax(site, days, dry_run=False):
         ## work flow for cleaning up illumina/analysis ##
```

Comment:
@senthil10 not related to this pool request but this might be a good occasion to improve. This `if` implies that if `archive` is not specified the projects need to be deleted. Maybe it is better to be explicit, and transform this `if-else` in a `if-elif-else` statement.